src/DevBetterWeb.Core/Specs/InactiveInvitationByEmailSpec.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace DevBetterWeb.Core.Specs
 {
-  public class InactiveInvitationByEmailSpec : Specification<Invitation>, ISingleResultSpecification
+  public partial class InactiveInvitationByEmailSpec : Specification<Invitation>, ISingleResultSpecification
```

Comment:
Why is this partial?

Reply:
excellent question. Not sure. Probably copy-paste error or autocomplete error